lib/badge.js

```diff
@@ -31,9 +34,17 @@ templateFiles.forEach(function(filename) {
     });
     compressSvg(untemplatedSvg, function(object) {
       if (object.error !== undefined) {
-        console.error('Template ' + filename + ': ' + object.error + '\n' +
-          '  Generated untemplated SVG:\n' +
-          '---\n' + untemplatedSvg + '---\n');
+        console.error(
+          'Template ' +
```

Comment:
Looks like a great use case for a format string.

Reply:
Is there a prettier option for prefer template string?